lombok.config

```diff
@@ -0,0 +1 @@
+lombok.addLombokGeneratedAnnotation = true
```

Comment:
From the lombok site:
> We advise against this; JDK9 breaks this annotation, and it's unlikely to ever get fixed.

Is there no longer-term resolution?

Reply:
@andrewwa-kainos That comment is for the property `lombok.addJavaxGeneratedAnnotation`